endpoints/getting-started-grpc/api_config.jwt.yaml

```diff
@@ -19,7 +19,7 @@
 #
 
 # The configuration schema is defined by service.proto file
-# https://github.com/googleapis/googleapis/blob/master/google/api/service.proto
+# https://github.com/googleapis/googleapis/blob/main/google/api/service.proto
```

Comment:
**N.B:** this is talking about a repo other than `nodejs-docs-samples`, and currently `404`s.

Maybe we should do one of the following? Feel free to use whichever one you'd prefer.
a) manually check that all these links work
b) not change anything outside of `nodejs-docs-samples` for now, and file a follow-up issue

Reply:
Thanks Ace! I fixed the offending links :)

Reply:
Thanks Ace! I fixed the offending links :)